test/utils.go

```diff
@@ -34,8 +35,28 @@ type (
 		*database.User
 		staticDB *database.DB
 	}
+	// ResponseWriter is a testing ResponseWriter implementation.
+	ResponseWriter struct {
```

Comment:
you should check out `httptest.NewRecorder()` maybe it's something you like and can adopt in one of your repos

Reply:
Sounds good, I'll take a look!

F/U: https://github.com/SkynetLabs/skynet-accounts/pull/152